src/client/containers/layout.js

```diff
@@ -47,6 +47,20 @@ class Layout extends React.Component {
 		this.renderNavHeader = this.renderNavHeader.bind(this);
 	}
 
+	handleDropdownToggle(newValue) {
+		if (this._forceOpen) {
```

Comment:
I would suggest renaming `forceOpen`to `keepMenuOpen` for clarity.


Is there a specific reason why you decided to use the component state for `menuOpen` but not for `forceOpen`?
I did try using the state, expecting that `handleDropdownToggle` would be called before `handleDropdownClick` setting the state had a chance to propagate, and sure enough that's what happened.
Was that a conscious decision you made for the same reason?

Reply:
it was, yeah
but since you suggested this, I began to think that this is inconsistent, so I am changing it